vng_api_common/polymorphism.py

```diff
@@ -59,7 +59,9 @@ def to_representation(self, instance) -> OrderedDict:
         return serializer.to_representation(instance)
 
     def to_internal_value(self, data) -> OrderedDict:
-        discriminator_value = data[self.discriminator_field]
+        discriminator_value = data.get(self.discriminator_field, '')
```

Comment:
Relying on vague/falsy values to detect if something is present or not is a bad practice. We're explicitly working around the case of partial updates where the discriminator field is missing, so that should be the first thing to check: `self.discriminator_field in data`. Only if it's NOT present, we fall back to the instance (if it's provided).

You might otherwise run into trouble when the field _is_ present, but it has an empty value (`''` or `None` or, much more likely: `False`). A discriminator with `False` and `True` as possible values is a possible case.

```suggestion
        discriminator_value = data[self.discriminator_field] if self.discriminator_field in data else getattr(instance, self.discriminator_field)
```